bitnami/grafana-operator/values.yaml

```
@@ -546,6 +546,12 @@ grafana:
   dashboardLabelSelectors:
     - matchLabels:
         app.kubernetes.io/instance: "{{ .Release.Name }}"
+  ## @param grafana.dashboardNamespaceSelector [object] This selects namespaces on the label.
```

Can you please change the wording a little bit? Take into account this sentence is going to be used in the README
```suggestion
  ## @param grafana.dashboardNamespaceSelector Watch for dashboards only in the Namespaces that have the specified namespace label
```